### DISCUSSION TITLE
Quick Experiments

### DISCUSSION THREAD

**Discussion by flawnn** [rodit/SnapMod#248] [General]
Hey guys,

anyone also tinkering around with quick experiments like me? Is there a way to get more than the ones just in the one XML file?

Greetings

**Comment by RevealedSoulEven** [comment]
What do you mean?

**Reply by flawnn** [reply]
Check the /data folder of Snapchat and go into the shared prefs (/data/data/com.snapchat.android/shared_prefs/) and check the file with QUICK_EXPERIMENTS in there. Not gonna write that name with the full extent cuz of Snapchat maybe googling this. There are some cool unreleased features there.

**Reply by RevealedSoulEven** [reply]
Hmmm seems interestinf

**Reply by ppperamaki** [reply]
What version of the app are you on? I'm on latest supported ver. and can't find the mentioned .xml file. Or were there other procedures to get quick experiments to show? Thanks.

**Comment by ppperamaki** [comment]
Hey, not sure how GitHub works and if you'll get a notif for my comment on a reply. I'm very interested in the topic and would love to help, make use of and find more unreleased features. However I've got the before-mentioned problem, which I'll paste here: What version of the app are you on? I'm on latest supported ver. and can't find the mentioned .xml file. Or were there other procedures to get quick experiments to show? Thanks.

**Reply by flawnn** [reply]
I am on latest beta. This file has been there for over 2 years now. It's in the directory I written down in my above post. Obviously you need proper root access for reading files in that directory. It's in the shared prefs of the Snapchat App Directory.

**Reply by ppperamaki** [reply]
Thanks for the reply. I've ran through the directories with many file managers, incl. ES, Termux which I've granted full file system and root access to. Still can't find it. Would you mind sharing the file in question on some platform or link it here? I'm intrigued by the contents and would love to integrate them into the app.

**Reply by flawnn** [reply]
https://hastebin.com/dosinojufi.csharp

here u go

**Reply by ppperamaki** [reply]
Alright, after some cross-referencing, I found that this exact file is under a different name for me (app_start_prefs), and I'm missing some lines compared to yours. Other than that it's the same. Wonder why it's like that. Thanks for the share.

**Comment by authorisation** [comment]
There used to be a Dark mode toggle there but Snapchat removed it

**Comment by RevealedSoulEven** [comment]
Why not re-adding it?

**Reply by flawnn** [reply]
Because they removed it from the app after XDA made an article about it.

**Reply by flawnn** [reply]
You could add the flag but that doesn't do shit

**Reply by authorisation** [reply]
it looked awful though

**Reply by RevealedSoulEven** [reply]
maybe, but the iOS dark mode is sexy

**Reply by Lucky-069** [reply]
Try using swift installer with snapchat. It gives you amoled dark mode!
